### DISCUSSION TITLE
adding anisotropic stiffness options (specifically to NorSand)

### DISCUSSION THREAD

**Discussion by jgiven100** [cb-geo/mpm#713] [Ideas]
As part of an on-going project, I am looking to add an anisotropic elastic stiffness tensor within our NorSand model.

I am planning to follow the work of Zhou (2015) which uses a scaling factor (`X`) such that the istropic and anisotropic compliance matrices result in equivalent volumetric strains. 

Is this a feature that others are interested in adding to CB-Geo?
* If yes, I can open an Issue for a formal feature discussion; 
* if no, I will work on this in a personal repo. 

Let me know what you all are thinking!

**Comment by kks32** [comment]
@jgiven100 Thank you! I think it would be great to have in CB-Geo MPM.

**Reply by debasismpt125** [reply]
yes, It will extend the capacity of CB-Geo to model sand.
